README.md

```diff
@@ -780,6 +780,7 @@ Advanced options can be configured as a parameter to the init() method or as env
 |useSSL             |EPSAGON_SSL                |Boolean|`true`       |Whether to send the traces over HTTPS SSL or not                                   |
 |traceCollectorURL  |EPSAGON_COLLECTOR_URL      |String |-            |The address of the trace collector to send trace to                                |
 |isEpsagonDisabled  |DISABLE_EPSAGON            |Boolean|`false`      |A flag to completely disable Epsagon (can be used for tests or locally)            |
+|keysToAllow        |EPSAGON_ALLOWED_KEYS       |Boolean|-            |Array of keys to automatically capture, label, and index in the trace              |
```

Comment:
It might be only me, but I'm not sure I understand what it does based on this readme